agent/config/config_test.go

```diff
@@ -380,3 +380,11 @@ func TestImageCleanupMinimumNumImagesToDeletePerCycle(t *testing.T) {
 		t.Errorf("Wrong value for NumImagesToDeletePerCycle: %v", cfg.NumImagesToDeletePerCycle)
 	}
 }
+
+func TestAWSVPCBlockInstanceMetadata(t *testing.T) {
```

Comment:
Please add a simple comment here?

Reply:
Isn't the name itself descriptive?

Reply:
Only if I know that this test was in the config package, it will have different meaning in `engine` package, where the test may be testing the process to block the metadata. Ignore my comments if we are supposed to have the context when reading the code.